docs/_docs/developers-guide/clients/jdbc-driver.adoc

```diff
@@ -16,10 +16,10 @@
 
 Apache Ignite is shipped with JDBC driver that allows processing of distributed data using standard SQL statements like `SELECT`, `INSERT`, `UPDATE`, or `DELETE` directly from the JDBC side. The name of the driver's class is `org.apache.ignite.jdbc.IgniteJdbcDriver`.
 
-This implementation of JDBC driver does not support:
-
-* Multiple endpoints
-* JDBC connection pools
+NOTE: The JDBC driver shipped with Apache Ignite version 3.2 has been completely reworked to add support for multiple endpoints and link:developers-guide/clients/overview#partition-awareness[partition awareness].
```

Comment:
shipped - > supplied ?
I'm not sure...

Reply:
Shipped fits the language of AI 2. Good alternative would be "available".